### ISSUE TITLE
No username was found bug after updating username recently.

### ISSUE THREAD

**Issue by Spat1o** [SkyCryptWebsite/SkyCrypt#2190]
**Describe the bug**
A "no username was found" error happens once entering new username. The new username has already been updated on api, because on other sites they work. SkyCryptError: No user with the name 'myusername' was found. This also happened to one of my friends and is annoying. You can only access it with the old username you previously had. Its actually also updating, me buying something also appears on skycrypt, however on the old username but not the new one.I have no idea if this is a known issue.

**Profile/Location**
On the site, for example my friend's username is Absorpy, however you can only access his skycrypt page via his old username (00122) the numbers are his username.

**Expected behavior**
Expect it to direct me to skycrypt page viewing profile information.

**Desktop (please complete the following information):**

- OS: Windows11
- Browser Chrome
- Veersion 121.0.6167.140

**Additional context**
Most if not all context has been described

**Comment by DuckySoLucky** [comment]
issue with an API we use for username revolving, cannot do anything here. temporary fix would be to use an UUID and then it should update an username

**Comment by Spat1o** [comment]
> issue with an API we use for username revolving, cannot do anything here. temporary fix would be to use an UUID and then it should update an username

Using a UUID does not update the username to the new one but it works as temporary fix, although it is a little bit annoying.
For example my friends is still 00122 https://sky.shiiyu.moe/stats/7dac4c64c50641c990dd0d4bd0077ec0

**Comment by DuckySoLucky** [comment]
yep, sadly we cannot do anything here. There's no other API that offers this kind of data without using multiple calls (which would slow down website even futher)

**Comment by Spat1o** [comment]
does that mean it will update the username after a time? If so, do you know how long that would take, because It has been about a week
